EIPS/eip-3675.md

```diff
@@ -75,6 +81,8 @@ Beginning with `TRANSITION_BLOCK`, a number of previously dynamic block fields a
 | **`nonce`** | `0x0000000000000000` |  |
 | **`ommers`** | `[]` | `RLP([]) = 0xc0`  |
 
+Beginning with `TRANSITION_BLOCK`, the validation of the block's **`extraData`** field changes: The length of the block's **`extraData`** **MUST** be less or equal to **`MAX_BYTES_PER_EXTRA_DATA`** bytes.
```

Comment:
```suggestion
Beginning with `TRANSITION_BLOCK`, the validation of the block's **`extraData`** field changes: The length of the block's **`extraData`** **MUST** be less or equal to **`MAX_EXTRA_DATA_BYTES`** bytes.
```
And here.

Reply:
suggest "less than or" rather than "less or"

Reply:
ACK! Done in db1c213.